src/main/resources/swagger/api-docs.yaml

```diff
@@ -295,3 +484,80 @@ definitions:
       - methodConfigurationName
       - entityType
       - entityName
+  ConfigurationIngest:
+    properties:
+      namespace:
+        type: string
+        description: Method configuration namespace
+      name:
+        type: string
+        description: Method configuration name
+      rootEntityType:
+        type: string
+        description: Root entity type
+      workspaceName:
+        type: string
```

Comment:
these all need to be Map[String, String], not string. Looks like in rawls, the methodRepoMethod and methodRepoConfig and workspaceName are strongly typed, even if we don't type them in orchestration.